deleting-clusters-and-cleaning-up.md

```diff
@@ -0,0 +1,39 @@
+# Deleting clusters and cleaning up
+
+## Deleting clusters
+Clusters managed by Cluster API can be deleted in the same way as any other Kubernetes object.
+
+To delete an individual cluster by name run:
+
+```bash
+kubectl delete cluster docker-cluster-one
+```
+This delete operation ensures that the cluster and all of the objects that form part of the cluster e.g. Machines, Cluster Infrastructure, are correctly deleted. 
+
+To delete all clusters managed by Cluster API run:
+
+```bash
+kubectl delete clusters --all -A
```

Comment:
Does this work cross-namespace? (I somehow remember that delete doesn't work with -A)

Just resolve if I'm wrong :)

Reply:
I think this usen't to work, but it does now.

Reply:
Maybe new kubectl version - again :D